zerver/tests/test_docs.py

```diff
@@ -106,6 +106,51 @@ def _test(
                     ['<meta name="robots" content="noindex,nofollow" />'], result
                 )
 
+    def test_api_doc_undocumented_endpoint(self) -> None:
+        # nasty mocking to get down into the KeyError
+        with mock.patch(
+            "zerver.lib.markdown.api_arguments_table_generator.get_openapi_parameters",
+            side_effect=KeyError,
+        ):
+            with self.assertLogs(level="ERROR"):
+                self.get_doc("/api/get-streams", subdomain="")
+        with mock.patch(
+            "zerver.lib.markdown.api_arguments_table_generator.get_openapi_parameters",
+            side_effect=KeyError(
+                ("parameters"),
+            ),
+        ):
+            self.get_doc("/api/get-streams", subdomain="")
+
+    def test_api_doc_nonjson_response(self) -> None:
```

Comment:
Same with this test.

Reply:
DONE: moved to PR https://github.com/zulip/zulip/pull/22057